pkg/core/ioutil/dump/dump.go

```diff
@@ -61,6 +61,12 @@ func YAML(stream io.Writer, body []byte) error {
 	return dumpYAML(stream, data)
 }
 
+// Empty returns no data so that in most cases including automated actions, no cluttering
+// of the console occurs.
+func Empty(stream io.Writer, body []byte) error {
```

Comment:
Why we would have empty function? Can we not have it :D?

Reply:
This doesn't seem to be used

Reply:
This one was among a couple changes i had created in anticipation of using in the method, but did not need it, so yes i will remove it.